go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -935,7 +952,7 @@ func (tsv *TabletServer) BeginStreamExecute(
 		return 0, nil, err
 	}
 
-	err = tsv.StreamExecute(ctx, target, sql, bindVariables, transactionID, options, callback)
+	err = tsv.StreamExecute(ctx, target, sql, bindVariables, transactionID, 0, options, callback)
```

Comment:
hmm.... should reserved id always be zero here? I'm confused

Reply:
no, `BeginStreamExecute` is not handling reserved connection, need code change.